### DISCUSSION TITLE
Output from one step (with ID) to another is empty.

### DISCUSSION THREAD

**Discussion by Duckle29** [community/community#25036] [Actions]
<p>Hey there, and thanks for the look :slight_smile:</p>
<p>I’m trying to wrap my head around github actions, and how it all works.</p>
<p>I’m currently trying to run a small script every time a PR is merged, with some specific files in it, and this seems to work.</p>
<p>Where I’m running in to issues is when I try to pass some values between steps.</p>
<p><a href="https://github.com/Duckle29/actions_test/runs/411393352#step:3:47" rel="nofollow noopener">From step 3 to step 4 works</a></p>
<p>However from <a href="https://github.com/Duckle29/actions_test/runs/411393352#step:4:85" rel="nofollow noopener">step 4 to step 5</a> doesn’t seem to work.</p>
<p>The entire workflow is <a href="https://github.com/Duckle29/actions_test/blob/master/.github/workflows/qmkpid.yml" rel="nofollow noopener">here</a></p>

**Comment by thboop** [comment, marked as answer]
<p>Hey <a class="mention" href="/duckle29">@duckle29</a> , I think you want </p>
<pre><code>${{steps.generate_qmkpid.outputs.output}}
</code></pre>
<p> <a href="https://github.com/Duckle29/actions_test/blob/master/.github/workflows/qmkpid.yml#L35" rel="nofollow noopener">here</a> </p>
<p>The following workflow worked for me below:</p>
<pre><code class="lang-auto">
steps:  
&amp;nbsp; - run: echo "::set-output name=foo::bar"  
&amp;nbsp; &amp;nbsp; &amp;nbsp;id: generate\_qmkpid  
&amp;nbsp; - run: echo ${{steps.generate\_qmkpid.outputs.foo}}

</code></pre>
<p>There’s another example located in the documentation here: <a href="https://github.com/actions/toolkit/blob/master/docs/commands.md#set-outputs" rel="nofollow noopener">https://github.com/actions/toolkit/blob/master/docs/commands.md#set-outputs</a></p>

**Comment by Duckle29** [comment]
<p>Thanks. That’s what I get for using ambiguous variable names :smiley:</p>
